components/Workspaces/InsightUpgradeModal.tsx

```diff
@@ -0,0 +1,112 @@
+              </p>
+            </section>
+
+            <section className="flex gap-8 justify-between w-full">
+              <Card className="basis-1/2 px-6 py-6 flex flex-col gap-6 justify-between">
+                <div className="flex items-center justify-between">
+                  <div className="flex flex-col gap-0.5">
+                    <h3 className="text-lg font-semibold">Free Workspace</h3>
+                    <p className="text-sm text-slate-500">Ideal for individuals</p>
+                  </div>
+
+                  <p className="text-4xl font-semibold">
+                    <span className="text-lg">$</span>0<span className="text-2xl text-slate-500">/mth</span>
+                  </p>
+                </div>
+
+                <ul className="flex flex-col gap-3">
+                  <ChecklistItem color="orange">Public only Workspaces</ChecklistItem>
+                  <ChecklistItem color="orange">
+                    Up to <span className="font-bold">10</span> contributors per insight page
+                  </ChecklistItem>
+                  <ChecklistItem color="orange">
+                    Up to <span className="font-bold">100</span> repositories per insight page
+                  </ChecklistItem>
+                </ul>
+
+                <Button
+                  variant="text"
+                  className="flex w-full py-3 justify-center !border !border-orange-600 text-orange-600"
+                >
+                  Your Plan
+                </Button>
+              </Card>
+
+              <Card className="basis-1/2 px-6 py-6 flex flex-col gap-6 justify-between">
+                <div className="flex items-center justify-between">
+                  <div className="flex flex-col gap-0.5">
+                    <h3 className="text-lg font-semibold">Pro Workspace</h3>
+                    <p className="text-sm text-slate-500">Ideal for teams </p>
+                  </div>
+
+                  <div className="flex flex-col gap-0.5 items-end">
+                    <p className="text-4xl font-semibold">
+                      <span className="text-xl">$</span>100<span className="text-2xl text-slate-500">/mth</span>
+                    </p>
+                    <p className="text-xs font-semibold text-slate-500">charged per Workspace</p>
+                  </div>
+                </div>
+
+                <ul className="flex flex-col gap-3">
+                  <ChecklistItem color="green">Private or public Workspaces, you choose!</ChecklistItem>
+                  <ChecklistItem color="green">
+                    Up to <span className="font-bold">1000</span> contributors per insight page
+                  </ChecklistItem>
+                  <ChecklistItem color="green">
+                    Up to <span className="font-bold">10,000</span> repositories per insight page
+                  </ChecklistItem>
+                </ul>
+
+                <Button variant="primary" className="py-3 flex justify-center">
```

Comment:
Refactor (nit): if it's centering text only, prefer `text-center`

```suggestion
                <Button variant="primary" className="py-3 text-center">
```

Reply:
`text-center` doesn't work, weirdly